src/Domain.LinnApps/Reports/PrefSupReceiptsReportService.cs

```diff
@@ -59,7 +61,9 @@ public string GetReportCurrencyValue(decimal currencyValue, string currency, dec
         public ResultsModel GetReport(DateTime fromDate, DateTime toDate)
         {
             var results = this.receiptRepository.FilterBy(
-                e => e.DateBooked >= fromDate && e.DateBooked <= toDate && e.Difference != 0);
+                e => e.DateBooked >= fromDate && e.DateBooked <= toDate && e.Difference != 0).OrderBy(
+                e => e.Difference,
+                OrderByDirection.Descending);
```

Comment:
is this faster than sorting outwith the repo? Does morelinq make the native db calls ?

Reply:
didn't need any speed difference but was hoping it would make its way into SQL (could alternatively stick it in to the View definition but that hides it a bit).

Reply:
if it's in the view then at least it's liable for database optimisation. if it's in the domain service then it's testable domain logic.  If morelinq doesn't make native code then it's untestable code and reliant for a third party library for no gain?